tests/nightly/nightly-autorun.sh

```diff
@@ -16,8 +16,10 @@
 
 # Get the latest code from vmware/vic repo
 cd ~/go/src/github.com/vmware/vic
-git fetch
+git clean -fd
+git fetch https://github.com/vmware/vic master
```

Comment:
I think we can just say `git fetch origin` instead of this. `git pull` on line 22 shouldn't be necessary either.